backend/otel/otel.go

```diff
@@ -428,6 +431,7 @@ func (o *Handler) submitProjectLogs(ctx context.Context, projectLogs map[string]
 				Type: kafkaqueue.MarkBackendSetup,
 				MarkBackendSetup: &kafkaqueue.MarkBackendSetupArgs{
 					ProjectID: projectIDInt,
+					Type:      model2.MarkBackendSetupTypeLogs,
```

Comment:
@ccschmitz we should only record setup events here if this is not a frontend log. the `SubmitFrontendConsoleMessage` function adds a source attribute to differentiate it

<img width="795" alt="Screenshot 2023-03-20 at 11 18 47 AM" src="https://user-images.githubusercontent.com/1351531/226431157-00c390bb-9ac2-4c78-87be-867c042cc7a6.png">

Reply:
Perfect - thanks, @Vadman97!